src/main/java/frc/team2412/robot/Commands/IndexerCommands/ProcessBalls.java

```diff
@@ -0,0 +1,29 @@
+package frc.team2412.robot.Commands.IndexerCommands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.Subsystems.ExampleSubsystem;
+import frc.team2412.robot.Subsystems.IndexerSubsystem;
+
+//This is an example command for this year. Make sure all commands extend CommandBase and they use take all dependencies(fields) through a constructor
+public class ProcessBalls extends CommandBase {

```

Comment:
"ProcessBalls" is a too-generic name.  Please use a more descriptive and specific name, like `IndexerInputFromFront` or `IndexerInputFromBack` or just `InputToIndexer` if it can handle receiving balls from either side at any time.  And consider using [command groups](https://docs.wpilib.org/en/latest/docs/software/commandbased/command-groups.html) instead of encoding a bunch of logic in the subsystem classs.